src/igvfd/audit/measurement_set.py

```diff
@@ -79,7 +79,38 @@ def audit_unspecified_protocol(value, system):
     if 'protocol' not in value:
         detail = (
             f'MeasurementSet {audit_link(path_to_text(value["@id"]),value["@id"])} '
-            f'are expected to specify the experimental protocol utilized for conducting '
+            f'is expected to specify the experimental protocol utilized for conducting '
             f'the assay on protocols.io.'
         )
         yield AuditFailure('missing protocol', detail, level='NOT_COMPLIANT')
+
+
+@audit_checker('MeasurementSet', frame='object')
+def audit_construct_libraries(value, system):
+    '''
+        audit_detail: Construct libraries linked to in a measurement set are expected to have the same library details.
+        audit_category: inconsistent construct library metadata
```

Comment:
Do you feel strongly about 'metadata'? Swapping in 'details' might be nice because it matches the property names being compared

Reply:
I agree with this change.